staging/src/k8s.io/client-go/tools/cache/delta_fifo.go

```diff
@@ -428,13 +459,29 @@ func (f *DeltaFIFO) Pop(process PopProcessFunc) (interface{}, error) {
 	}
 }
 
+const (
+	resyncEndKey = "DeltaFifo::internal::resyncEnd"
+	relistEndKey = "DeltaFifo::internal::relistEnd"
+)
+
 // Replace will delete the contents of 'f', using instead the given map.
 // 'f' takes ownership of the map, you should not reference the map again
 // after calling this function. f's queue is reset, too; upon return, it
 // will contain the items in the map, in no particular order.
 func (f *DeltaFIFO) Replace(list []interface{}, resourceVersion string) error {
+	if f.RelistRunning != nil {
```

Comment:
No, Resync is called on resyncs. Replace is called for relists. But please do doublecheck my research!